.yarn/sdks/typescript/lib/tsserver.js

```diff
@@ -96,9 +114,9 @@ const moduleWrapper = tsserver => {
         parsedMessage != null &&
         typeof parsedMessage === `object` &&
         parsedMessage.arguments &&
-        parsedMessage.arguments.hostInfo === `vscode`
+        typeof parsedMessage.arguments.hostInfo === `string`
       ) {
-        isVSCode = true;
+        hostInfo = parsedMessage.arguments.hostInfo;
```

Comment:
Why is this file changing whout a change in the lockfile?

Reply:
I believe because I re-ran `yarn dlx @yarnpkg/pnpify --sdk base`, which will have downloaded the latest version of that package. See eg https://github.com/yarnpkg/berry/blob/master/packages/yarnpkg-pnpify/sources/sdks/base.ts#L37

Reply:
Thanks